src/app/loaders/express/index.ts

```diff
@@ -131,10 +133,19 @@ const loadExpressApp = async (connection: Connection) => {
   // New routes in preparation for API refactor.
   app.use('/api', ApiRouter)
 
+  // serve static assets
   app.use(express.static(path.resolve('dist/frontend'), { index: false }))
   app.use('/public', express.static(path.resolve('dist/angularjs')))
-  app.get('/old/', HomeController.home)
 
+  // Requests for known static asset patterns which were not served by
+  // the static handlers above should return 404s
+  app.get(/^\/(public|static)\//, catchNonExistentRoutesMiddleware)
+
+  // Requests for root files (e.g. /robots.txt or /favicon.ico) that were
+  // not served statically above will also return 404
+  app.get(/^\/[^/]+\.[a-z]{2,}$/, catchNonExistentRoutesMiddleware)
```

Comment:
No objections per se but just wondering why we need to be so specific with the filename format - are there no legitimate cases in future where we could have:
- filename with no extension
- filename with single-character extension